com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/internal/ash/AshFrameHandler.java

```diff
@@ -429,6 +432,8 @@ private synchronized boolean sendNextFrame() {
             // Nothing to send
             return false;
         }
+        int sequenceNumber = nextEzspSequenceNumber.getAndIncrement() & 0xff;
```

Comment:
Since ```sequenceNumber``` doesn't appear to be used at all (other than in the next line) it might be better not to create this variable and directly set the sequence number?

```
nextFrame.setSequenceNumber(nextEzspSequenceNumber.getAndIncrement() & 0xff);
```